guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -80,27 +88,69 @@ The `ece-tools` package provides a `docker:build` command to generate the Docker
     ```bash
     docker-compose run build cloud-build
     ```
-    
+
     * Deploy Magento in the Docker container:
-    
+
     ```bash
     docker-compose run deploy cloud-deploy
     ```
 
-1. Access your local Magento Cloud template by opening one of the following secure URLs in a browser:
- 
-    -  [`http://localhost:8080`](http://localhost:8080){:target="\_blank"}
-    -  [`https://localhost`](https://localhost){:target="\_blank"}
+1.  Open the `http://localhost:8080` URL in a browser to access your local Magento Cloud template.
```

Comment:
Do we need both instead of this one? The reason I ask is one is HTTP and the other is HTTPS. Maybe there is a reason Oleh mentioned both.